yeaseul/src/components/Store.tsx

```diff
@@ -0,0 +1,180 @@
+import styled from "styled-components";
+
+interface Props {
+  title: string;
+  summary: string;
+  imgSrc: string;
+  grade: string;
+  grade2: string;
+  menuSrc0: string;
+  menuName0: string;
+  price0: string;
+  menuSrc1: string;
+  menuName1: string;
+  price1: string;
+  menuSrc2: string;
+  menuName2: string;
+  price2: string;
+  menuSrc3: string;
+  menuName3: string;
+  price3: string;
+}
```

Comment:
ㅋㅋㅋㅋㅋㅋ 고생하셨습니다 👍 👍 
이런 경우에는 객체를 받으면 더 편할 거 같지 않으세요?

```suggestion

interface Menu {
  src: string;
  name: string;
  price: string;
}

interface Props {
  title: string;
  summary: string;
  imgSrc: string;
  grade: string;
  grade2: string;
  menu1: Menu;
  menu2: Menu;
  menu3: Menu;
}
```

혹은 여기서 더 나가서 배열로도 받을 수 있겠구요 ~

```
menus: Menu[];
```

반복되는 작업을 어떻게 줄이면서 직관적으로 풀어나갈지 고민해 보시면 좋을 거 같아요!

Reply:
오.... 이런 방법이...🫢